thriftpy2/http.py

```diff
@@ -99,6 +99,24 @@ def __init__(self, handler):
         self.handler = handler
 
 
+class THttpHeaderFactory(object):
+    """Default header factory return no custom headers
+    """
+    def __init__(self, headers=None):
+        """Initialize a header factory
+        @param headers(dict) a dictionary of static headers the factory generates
+        """
+        self.__headers = headers
+
+
+    def __enter__(self):
```

Comment:
Why there is a enter function here?

Reply:
That is on me, I was having problems when using a context and thought this was the fix, turns out I was wrong. Updated